.goreleaser.yml

```diff
@@ -59,9 +59,8 @@ brews:
     dependencies:
       - name: "go"
         type: build
-    commit_author:
-      name: "Aaron Aldrich"
-      email: "aaldrich@equinix.com"
+    test: |
+      system "#{bin}/metal -v"
```

Comment:
Adds test permanently to brew formula as noted in [equinix/homebrew-tap#2](https://github.com/equinix/homebrew-tap/pull/2)